packages/cli/src/config/parseStoreConfig.ts

```diff
@@ -151,22 +169,29 @@ export interface StoreUserConfig {
    * The key is the table name (capitalized).
    *
    * The value:
-   *  - `SchemaType | userType` for a single-value table (aka ECS component).
+   *  - abi or user type for a single-value table (aka ECS component).
```

Comment:
not part of this PR, but I think we should remove `aka ECS component`. Having a single value is not the distinguishing factor for an ECS component, since components can have multiple (related) values (eg. `PositionComponent = {x: int32, y: int32 }`). The distinguishing factor for ECS components vs regular tables would rather be having a single key vs a composite key.